source/MRMesh/MRGcodeProcessor.cpp

```diff
@@ -53,12 +56,13 @@ std::vector<MR::GcodeProcessor::MoveAction> GcodeProcessor::processSource()
     return res;
 }
 
-GcodeProcessor::MoveAction GcodeProcessor::processLine( const std::string_view& line )
+GcodeProcessor::MoveAction GcodeProcessor::processLine( const std::string_view& line, std::vector<Command> & commands )
 {
     if ( line.empty() )
         return {};
 
-    auto commands = parseFrame_( line );
+    commands.clear();
```

Comment:
mb move in parseFrame_(...)?

Reply:
parseFrame_ is declared as
```
parseFrame_( const std::string_view& frame, std::vector<Command> & outCommands );
```
So it is good that it can add command to not-empty vector